src/Apps/Auctions/Components/CuritorialRailsTabBar.tsx

```diff
@@ -16,17 +18,27 @@ export const CuritorialRailsTabBar: React.FC<CuritorialRailsTabBarProps> = ({
   viewer,
 }) => {
   const showWorksForYouTab = !!viewer.followedArtistsInAuction?.counts?.total
+  const { user } = useSystemContext()
 
   return (
     <Tabs mb={4}>
       {showWorksForYouTab && (
-        <Tab name="Lots for You">
+        <Tab name="Works for You">
```

Comment:
@dblandin changed this back to "Works for You" since we're adding the "Lots for You" from Homepage (different functionality)